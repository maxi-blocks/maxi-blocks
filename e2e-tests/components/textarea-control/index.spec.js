/**
 * WordPress dependencies
 */
import { createNewPost, pressKeyWithModifier } from '@wordpress/e2e-test-utils';
/**
 * Internal dependencies
 */
import {
	openSidebarTab,
	getBlockStyle,
	insertMaxiBlock,
	updateAllBlockUniqueIds,
} from '../../utils';

describe('TextareaControl', () => {
	it('Check textarea control', async () => {
		await createNewPost();
		await insertMaxiBlock(page, 'Image Maxi');
		await updateAllBlockUniqueIds(page);
		const accordionPanel = await openSidebarTab(
			page,
			'style',
			'hover effect'
		);

		await accordionPanel.$$eval(
			'.maxi-hover-effect-control .maxi-settingstab-control button',
			buttons => buttons[2].click()
		);

		await page.$eval(
			'.maxi-hover-effect-control .maxi-base-control__field textarea',
			select => select.focus()
		);
		await pressKeyWithModifier('primary', 'a');
		await page.keyboard.press('Backspace');
		await page.keyboard.type('Testing everything works correctly!');

		const expectText = await page.$eval(
			'.maxi-hover-effect-control .maxi-base-control__field textarea',
			expectHtml => expectHtml.innerHTML
		);

		expect(expectText).toMatchSnapshot();

		expect(await getBlockStyle(page)).toMatchSnapshot();
	});
});
