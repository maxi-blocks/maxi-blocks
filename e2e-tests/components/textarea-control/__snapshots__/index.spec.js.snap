// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextareaControl Check textarea control 1`] = `"Testing everything works correctly!"`;

exports[`TextareaControl Check textarea control 2`] = `"<style>body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1{ justify-content: center;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1{ justify-content: center;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper{ justify-content: center;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper{ justify-content: center;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper img,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper img{ width: 100%;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper img,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper img,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper img,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper img,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 .maxi-image-block-wrapper img{ width: 100%;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 figcaption,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 figcaption{ width: 100%; margin-top: 1em;}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption{ width: 100%; margin-top: 1em;}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption{ color: var(--maxi-light-link,rgba(var(--maxi-light-color-4,255,74,23),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption{ color: var(--maxi-light-link,rgba(var(--maxi-light-color-4,255,74,23),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:hover{ color: var(--maxi-light-link-hover,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:hover{ color: var(--maxi-light-link-hover,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active{ color: var(--maxi-light-link-active,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active{ color: var(--maxi-light-link-active,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active span{ color: var(--maxi-light-link-active,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:active span{ color: var(--maxi-light-link-active,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited{ color: var(--maxi-light-link-visited,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited{ color: var(--maxi-light-link-visited,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited span{ color: var(--maxi-light-link-visited,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited span{ color: var(--maxi-light-link-visited,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  a figcaption.maxi-image-block__caption:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  a figcaption.maxi-image-block__caption:visited{ color: var(--maxi-light-link,rgba(var(--maxi-light-color-4,255,74,23),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  a figcaption.maxi-image-block__caption:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  a figcaption.maxi-image-block__caption:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  a figcaption.maxi-image-block__caption:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  a figcaption.maxi-image-block__caption:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  a figcaption.maxi-image-block__caption:visited{ color: var(--maxi-light-link,rgba(var(--maxi-light-color-4,255,74,23),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited:hover{ color: var(--maxi-light-link-hover,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 a figcaption.maxi-image-block__caption:visited:hover{ color: var(--maxi-light-link-hover,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a{ color: var(--maxi-light-link,rgba(var(--maxi-light-color-4,255,74,23),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a{ color: var(--maxi-light-link,rgba(var(--maxi-light-color-4,255,74,23),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:hover{ color: var(--maxi-light-link-hover,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:hover{ color: var(--maxi-light-link-hover,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active{ color: var(--maxi-light-link-active,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active{ color: var(--maxi-light-link-active,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active span{ color: var(--maxi-light-link-active,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:active span{ color: var(--maxi-light-link-active,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited{ color: var(--maxi-light-link-visited,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited{ color: var(--maxi-light-link-visited,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited span{ color: var(--maxi-light-link-visited,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited span,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited span{ color: var(--maxi-light-link-visited,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  figcaption.maxi-image-block__caption a:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  figcaption.maxi-image-block__caption a:visited{ color: var(--maxi-light-link,rgba(var(--maxi-light-color-4,255,74,23),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  figcaption.maxi-image-block__caption a:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  figcaption.maxi-image-block__caption a:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  figcaption.maxi-image-block__caption a:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  figcaption.maxi-image-block__caption a:visited,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1.block-editor-block-list__block  figcaption.maxi-image-block__caption a:visited{ color: var(--maxi-light-link,rgba(var(--maxi-light-color-4,255,74,23),1));}body.maxi-blocks--active .edit-post-visual-editor .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited:hover{ color: var(--maxi-light-link-hover,rgba(var(--maxi-light-color-6,201,52,10),1));}body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xl\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"l\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"m\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"s\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited:hover,body.maxi-blocks--active .edit-post-visual-editor[maxi-blocks-responsive=\"xs\"] .maxi-block.maxi-block--backend.image-maxi-1 figcaption.maxi-image-block__caption a:visited:hover{ color: var(--maxi-light-link-hover,rgba(var(--maxi-light-color-6,201,52,10),1));}</style>"`;
